azure_license-server/security_group_rules.tf

```diff
@@ -0,0 +1,50 @@
+variable "rg_name" {}
+variable "rg_location" {}
+
+resource "azurerm_network_security_group" "security_group" {
```

Comment:
I'm testing this and seeing the following error:
```
│ Error: creating/updating Network Security Group: (Name "license-server-security-group" / Resource Group "sentieon-licsrvr"): \
  network.SecurityGroupsClient#CreateOrUpdate: Failure sending request: StatusCode=400 -- \
  Original Error: Code="SecurityRuleParametersMissingForPortRanges" \
    Message="Required security rule parameters are missing for security rule with Id: /subscriptions/3e1e983c-58aa-4db9-ac6e-18392984d537/resourceGroups/sentieon-licsrvr/providers/Microsoft.Network/networkSecurityGroups/license-server-security-group/securityRules/AllowLicenseServerInboundSSH. Security rule must specify either SourcePortRange or SourcePortRanges." \
    Details=[]
│
│   with module.security_group.azurerm_network_security_group.security_group,
│   on security_group/security_group_rules.tf line 4, in resource "azurerm_network_security_group" "security_group":
│    4: resource "azurerm_network_security_group" "security_group" {
```

Reply:
The `AllowLicenseServerInboundSSH` and `AllowLicenseServerICMP` security rules may need to be updated.

Reply:
Will take a look and retry the deployment